addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html

```diff
@@ -53,6 +53,11 @@ <H2>CSRF Token</H2>
 <H2>.env Information Leak</H2>
 Checks for web accessible .env files which may leak sensitive information 
 (such as usernames, passwords, API or APP keys, etc.).
+Environment files come in many flavors but mostly they are KEY=VALUE formatted. <br>
+NGINX returns them as binary/octet-stream content-type Apache just returns the text
```

Comment:
This could be reworded so that's clear the scan rule is doing these checks (this just seems to be describing the behaviour of servers).

Reply:
tried